atmo/jobs/models.py

```diff
@@ -226,3 +213,53 @@ def delete(self, *args, **kwargs):
 
     def get_results(self):
         return self.provisioner.results(self.identifier, self.is_public)
+
+
+@python_2_unicode_compatible
+class SparkJobRun(EditedAtModel):
+
+    spark_job = models.ForeignKey(
+        SparkJob,
+        on_delete=models.CASCADE,
```

Comment:
So I'm wondering if we should not have this cascade but set to NULL instead to be able to keep records of job runs, even if the users delete the jobs in the frontend. Do you think we can find a pattern to clean up differently?